### PR TITLE
Consignment QA

### PR DESCRIPTION
**Pull request by orta**
Working on - https://github.com/artsy/consignments/issues/89

This is not everything, but I'd like to get it in so that I can look at a few small bugs separate from this.

Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Consignments/Components/__tests__/DoneButton-tests.tsx`
- `src/lib/Components/Consignments/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/CloseButton-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Consignments/Components/__tests__/DoneButton-tests.tsx`
- `src/lib/Components/Consignments/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/CloseButton-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Consignments/Components/__tests__/DoneButton-tests.tsx`
- `src/lib/Components/Consignments/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/CloseButton-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Consignments/Components/__tests__/DoneButton-tests.tsx`
- `src/lib/Components/Consignments/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/CloseButton-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Consignments/Components/__tests__/DoneButton-tests.tsx`
- `src/lib/Components/Consignments/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/CloseButton-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Consignments/Screens/Artist.tsx [hunk 1]
```diff
@@ -66,11 +66,19 @@ export default class Artist extends React.Component<Props, State> {
   }
 
   render() {
+    const isPad = Dimensions.get("window").width > 700
```
Comment (ashfurrow) [inline src/lib/Components/Consignments/Screens/Artist.tsx]: We should really factor this logic out into a helper function sometime.

Comment (orta) [reply src/lib/Components/Consignments/Screens/Artist.tsx]: Yeah, there's the onLayout technique, but it's a lot of code and doubles the render pass :-/

**Pull request merged**